octobot_tentacles_manager/managers/tentacles_init_files_manager.py

```diff
@@ -59,14 +59,30 @@ def _add_tentacle_to_tentacle_type_init_file(init_content, tentacle, init_file):
             init_file_w.write(init_content)
 
 
-def _remove_tentacle_from_tentacle_type_init_file(init_content, tentacle, init_file):
-    if init_content:
+def _remove_tentacle_from_tentacle_type_init_file(tentacle, init_file):
+    init_content_lines = []
+    if isfile(init_file):
+        # load import file
+        with open(init_file, "r") as init_file_r:
+            init_content_lines = init_file_r.readlines()
+    if init_content_lines:
         # remove import line
-        to_remove_line = f"{get_tentacle_import_block(tentacle)}\n"
-        if to_remove_line in init_content:
-            init_content = init_content.replace(to_remove_line, "")
+        to_remove_lines_count = len(get_tentacle_import_block(tentacle).split("\n")) - 2
```

Comment:
It would be better to use https://docs.python.org/3/library/os.html#os.linesep instead of `\n`.

Reply:
It doesn't work since the splitted lines are python string and not read lines

Reply:
![image](https://user-images.githubusercontent.com/10454670/77637840-8fe15000-6f56-11ea-8029-1d4d908491be.png)

Reply:
![image](https://user-images.githubusercontent.com/9078616/77638069-e2227100-6f56-11ea-8e65-6a18ee8c09e5.png)
(windows)

Reply:
Ok then I think it should be better to declare this `\n' constant somewhere